app/models/ability.rb

```diff
@@ -312,6 +312,7 @@ def package_abilities
              fetch_added_quantity], Package
       can %i[show create update destroy], PackageSet
       can %i[index], Restriction
+      can %i[index], PackagesInventory
```

Comment:
Just making a note on this:

The `can_manage_packages` , is available for the roles Reviewer, Supervisor, Order Fulfilment and Order Administrator, which I feel is wrong

- Would it be better to have `can_manage_packages` permissions only for the Order Fulfilment and Order Administrator roles?
OR
- Would it be better to have a separate permission for `PackagesInventory`?

cc: @steveyken

Reply:
My 2 cents:

I agree that the Reviewer shouldn't have this access. Supervisor is all powerful, so no issues there.

Regarding a permission for the inventory:
- It _would_ be better, but I think it'll be closely tied to `can_manage_package`, they'll probably go hand-in-hand all the time. Given that it's read-only for now, I'm fine keeping it like this
- The inventory really works as an internal mechanism, I'm still a bit weary about exposing it. I'm afraid a permission would encourage people to add more functionality on the controller.

Just food for thought, it's a good question

Reply:
Good issue to raise. I'd like to rename Reviewer to "Offer Reviewer" and Supervisor to "Offer Administrator". These roles only need the ability to manage packages directly related to Offers (via items). We can scope CanCanCan rules to do that.

The Stock Fulfilment, Stock Administrator, Order Fulfillment, Order Administrator roles can have full can_manage_packages permissions and given that PackagesInventories is tightly bound to Packages I think it is fine for that to cover access to both tables so what is done above is fine.